modules/threat_hunt_rules/detail-layout.json

```diff
@@ -181,34 +181,51 @@
                                                                                             {
                                                                                                 "fields": [
                                                                                                     {
-                                                                                                        "name": "qRadar",
+                                                                                                        "name": "sTHFortiSIEM",
```

Comment:
sTHFortiSIEM? what is this name all about?

Reply:
Added tooltips